source/DistanceMap.cpp

```diff
@@ -41,7 +41,7 @@ DistanceMap::DistanceMap(const PlayerInfo &player, const System *center)
 {
 	if(!player.Flagship())
 		return;
-	
+
```

Comment:
Please dont get me wrong, but your pull will - as far as I know - never be merged if you dont fix that in this pull too. It is good to change it for the future, but if the author would merge this pull, all your trimming would be applied to the master branch.
I did not even look at 2 or 3 files of your initial commit because the diff was not loaded initially (because of that many changes).
So please fix this for your existing pull too.